x-pack/plugins/maps/public/classes/layers/solution_layers/security/security_index_pattern_utils.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+
+import minimatch from 'minimatch';
+import { SimpleSavedObject } from 'src/core/public';
+import { getIndexPatternService, getUiSettings } from '../../../../kibana_services';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { IndexPatternSavedObjectAttrs } from '../../../../../../../../src/plugins/data/public/index_patterns/index_patterns/index_patterns';
+
+const SIEM_DEFAULT_INDEX = 'siem:defaultIndex';
```

Comment:
This just changed and is now `securitySolution:defaultIndex`

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/84207192-04b02a80-aa6e-11ea-9f47-cc3ca98fff52.png" />
</p>

Reply:
stop changing things...

Reply:
I think @XavierM shares that same sentiment... 😅